_posts/2020-07-21-tips-and-tricks-for-running-strimzi-with-kubectl.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title:  "Tips & Tricks for running Strimzi with kubectl"
+date: 2020-07-21
+author: jakub_scholz
+---
+
+When running Apache Kafka on Kubernetes with Strimzi, one of the tools you might use very often is `kubectl`.
+You can use it for controlling your Kubernetes cluster and its resources.
+And because Strimzi is using Custom Resources to extend the Kubernetes APIs, it can use it also with Strimzi.
+In this blog post, we will have a look at some tips and tricks how to make it easier.
+
+<!--more-->
+
+## Working with Kubernetes resources
+
+`kubectl` can be used for all the basic tasks with the Strimzi custom resources.
+You can use all `kubectl` commands such as `get`, `describe`, `edit`, or `delete` coupled with the resource type.
+So for example `kubectl get kafkatopics` will get you a list of all Kafka topics or `kubectl get kafkas` to get all Kafka clusters.
+
+Most people know that for all of the resource types, you can use both singular and plural.
```

Comment:
Unsure if this would potentially shame someone who doesn't know this. Perhaps it would be better to be informative, i.e

```suggestion
When referencing a resource types, you can use both singular and plural.
```